test/testdrive/explain-timestamps.td

```diff
@@ -0,0 +1,24 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ set-regex match=(\d{13}|u\d{1,3}) replacement=<REDACTED>
+
+> CREATE TABLE t1 (a INT);
+
+> CREATE TABLE t2 (a INT);
+
+# Strict serializable doesn't look at every object in the same time domain
```

Comment:
Am I correct that this comment is only true from EXPLAIN's point of view? sequence_peek doesn't about time domain, and neither does determine_timestamp.

Reply:
I think this is half true from SELECT's point of view too. Every ID in the timedomain is passed into `determine_timestamp`, but in Strict Serializable mode we don't look at them except to determine the timeline.